src/main.ts

```diff
@@ -21,25 +25,69 @@ if (cli.input.length === 0) {
 }
 
 let port = DEFAULT_PORT
+let rest_port = 9999
 if (cli.input.length > 1) {
 	port = Number(cli.input[1])
 	if (isNaN(port)) {
 		cli.showHelp(1)
 	}
+	if (cli.input.length > 2) {
+		rest_port = Number(cli.input[2])
+		if (isNaN(rest_port)) {
+			cli.showHelp(1)
+		}
+	}
 }
 
 console.log('Starting')
 
 const client = new CompanionSatelliteClient({ debug: true })
 const devices = new DeviceManager(client)
+const server = new RestServer(client, rest_port)
 
 client.on('log', (l) => console.log(l))
 client.on('error', (e) => console.error(e))
 
+client.on('ipChange', (newIP) => {
+	fs.access('/boot/satellite-config', (err: any) => {
```

Comment:
Im not liking how manual this process is, but Im not too sure on how to improve it.

Perhaps it would be easier if the config was switched out to being an `ini` file? 
Or maybe that would overcomplicate further.

It would be good to have this pulled out into a function, something like:
```function updateEnvironmentFile(filePath: string, changes: Record<string, string>):void ```
so that when we add more fields it can be easily reused.